src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -89,6 +122,16 @@ function CardSection() {
                 )}
                 {isEmptyObject(defaultCard?.accountData) && <CardSectionDataEmpty />}
```

Comment:
Could we have `isEmptyObject(defaultCard?.accountData) === true` and the following condition `billingStatus?.isRetryAvailable !== undefined`  ? Because that would mean on the UI the user see the empty card and a retry button.

Reply:
In theory it might be possible if BE was not responding or something but Im not 100% sure. Normally its not possible. In order to get a retry button user needs to get an error first that allows him to retry payment. In order to get such error he has to add a card first